e2e-tests/README.md

```diff
@@ -2,15 +2,20 @@
 
 This is an automation tests folder for Virtual Visits sample, which contains all UI tests.
 
-## Build and Run End to End tests
+## Build and Run End to End tests locally
 
-- To install supported browsers, use the following command:
+- Before running the tests make sure that the environment variable for connection string is set. [Learn more about environment variables and how to set them.](https://github.com/Azure-Samples/communication-services-virtual-visits-js#environment-variables)
+
+- To install dependencies, use the following command:
 
 ```
+npm install
 npx playwright install
 
 ```
 
+For more information about browsers, check the [Official Playwright Documentation for Browsers](https://playwright.dev/docs/browsers)
+
 - To build and run the tests, use the following commands:
 
 ```
```

Comment:
I ran `npm run test` command and failed the test in the very first time. Is this result expected?

![image](https://user-images.githubusercontent.com/94404633/175439862-dacaf596-a430-4109-8824-db35eb04a085.png)

![image](https://user-images.githubusercontent.com/94404633/175439906-427cb018-4c28-4627-99c6-4238281744c4.png)
![image](https://user-images.githubusercontent.com/94404633/175439921-7110d40a-2002-4de7-8b08-ca71329e1abe.png)